tls-client/mbed_app.json

```diff
@@ -2,8 +2,36 @@
     "macros": [
         "MBEDTLS_USER_CONFIG_FILE=\"mbedtls_entropy_config.h\""
     ],
+    "config": {
+        "network-interface":{
+            "help": "options are ETHERNET,WIFI_ESP8266,WIFI_ODIN,WIFI_RTW,MESH_LOWPAN_ND,MESH_THREAD",
+            "value": "ETHERNET"
+        },
+        "esp8266-tx": {
+            "help": "Pin used as TX (connects to ESP8266 RX)",
+            "value": "PTD3"
+        },
+        "esp8266-rx": {
+            "help": "Pin used as RX (connects to ESP8266 TX)",
+            "value": "PTD2"
+        },
+        "esp8266-debug": {
+            "value": true
+        },
+        "wifi-ssid": {
+            "value": "\"SSID\""
+        },
+        "wifi-password": {
+            "value": "\"Password\""
+        }
+    },
     "target_overrides": {
-        "UBLOX_EVK_ODIN_W2": {
+        "*": {
+            "target.features_add": ["NANOSTACK"],
```

Comment:
As mentioned before, this is causing build failures with ARMCC

Reply:
Then we have no other option than use the `.mbedignore`.